fero/analysis.py

```diff
@@ -439,8 +489,9 @@ def make_prediction(
     def make_prediction_serial(
         self, prediction_data: Union[pd.DataFrame, List[dict]]
     ) -> Union[pd.DataFrame, List[dict]]:
-        """Makes a prediction from the provided data using the most recent trained model for the analysis. This
-        computes predictions one row at a time. Therefore, while it is slower than `make_prediction`, this method
+        """Make a prediction from the provided data using the most recent trained model for the analysis.
```

Comment:
I think we no longer need this, since everyone is on v2 models now.

Reply:
Got it! removed this function, tests of it, and references to it in other comments.